targets/targets.json

```diff
@@ -8774,9 +8774,9 @@
         "deliver_to_target": "NU_PFM_M2351_NPSA_NS",
         "delivery_dir": "TARGET_NUVOTON/TARGET_M2351/TARGET_M23_NS/TARGET_NU_PFM_M2351_NPSA_NS/TARGET_NU_PREBUILD_SECURE",
         "mbed_rom_start"        : "0x0",
-        "mbed_rom_size"         : "0x40000",
+        "mbed_rom_size"         : "0x10000",
         "mbed_ram_start"        : "0x20000000",
-        "mbed_ram_size"         : "0x8000"
+        "mbed_ram_size"         : "0x2000"
```

Comment:
Should we add `"target.boot-stack-size"            : "0x600"` here and also `"target.device_has_remove": ["SERIAL", "SERIAL_ASYNCH", "SERIAL_FC", "STDIO_MESSAGES"],`?

Reply:
Yes, added.